### PR TITLE
Adding functions to read/write 'La Palma' and 'Solarnet' cubes

### PR DESCRIPTION
**Pull request by cdiazbas**
#1 Functions to read/write 'La Palma' cubes have been added.
I have re-written some parts to make it clear and more organised.

#2 Functions to read/write 'Solarnet' cubes have been added.
Also to get time, wavelength, coordinates, etc.

### CODE REVIEW

File: io/solarnet.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    """
+    hour = number//3600.
+    minute = (number%3600.)//60.
+    seconds = (number-(hour*3600.+minute*60))/1.
+    string_output = '{0}:{1}:{2:09.6f}'.format(int(hour),int(minute),seconds)
+    return string_output
+
+
+# ========================================================================
+def get_time(filename, fulltime=False, timeformat=False):
```
Comment (grviss) [inline io/solarnet.py]: Maybe change `timeformat` keyword to `utc` for clarity.

File: io/solarnet.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+    if timeformat is True:
+        time_output_ = np.array([seconds2string(number) for number in time_output.reshape(time_output.size)])
+        time_output = time_output_.reshape(time_output.shape)
+
+    return time_output
+
+
+# ========================================================================
+def get_extent(filename, timeFrame=0):
```
Comment (grviss) [inline io/solarnet.py]: Keyword `timeFrame` is currently not used in the function.

File: io/solarnet.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    Authors: Carlos Diaz (ISP/SU 2019)
+    """
+    io = fits.open(filename)
+    extent_output = [io[1].data[0][0][0,0,0,0,0],io[1].data[0][0][0,0,1,1,0],
+            io[1].data[0][0][0,0,0,0,1],io[1].data[0][0][0,0,1,1,1]]
+
+    return extent_output
+
+# ========================================================================
+def get_coord(filename, pix_x,pix_y,timeFrame=0):
```
Comment (grviss) [inline io/solarnet.py]: Keyword `timeFrame` is currently not used in the function.

Comment (cdiazbas) [reply io/solarnet.py]: Thanks. Solved in last commit.

File: io/solarnet.py (hunk 2 above)
Comment (cdiazbas) [reply io/solarnet.py]: Thanks. Solved in last commit.

File: io/solarnet.py (hunk 1 above)
Comment (cdiazbas) [reply io/solarnet.py]: Thanks. Solved in last commit.

**Pull request merged**